hackathon_site/dashboard/frontend/src/components/general/Navbar/Navbar.test.tsx

```diff
@@ -13,16 +12,18 @@ describe("<Navbar />", () => {
         ["Teams", "/teams"],
         ["Reports", "/reports"],
         ["Inventory", "/inventory"],
-        [`Cart (${cartQuantity})`, "/cart"],
+        [`Cart (0)`, "/cart"],
     ];
 
+    const handleLogoutSpy = jest.fn();
+
     pagesAndPaths.map(([label, path]) => {
         it(`Adds the active class to ${label} when on ${path}`, () => {
             const { getByText, container } = render(
-                <Navbar cartQuantity={cartQuantity} pathname={path} />
+                <UnconnectedNavbar pathname={path} logout={handleLogoutSpy} />
```

Comment:
you can keep this as Navbar component, was there a reason why you changed it to UnconnectedNavbar?

Reply:
@YuYing-Liang I get this error when I use the Navbar `Type '{ pathname: string; cartQuantity: number; }' is not assignable to type 'IntrinsicAttributes & Omit<NavBarProps, "logout" | "pathname">'.   Property 'pathname' does not exist on type 'IntrinsicAttributes & Omit<NavBarProps, "logout" | "pathname">'.`

Reply:
Yup, that's because navbar doesn't have logout property, you can remove that, the original code didn't have logout property either

Reply:
The error above comes when I use the original code, ` <Navbar cartQuantity={cartQuantity} pathname={path} />`

Reply:
Lol so you removed cartQuantity from the props remember? So now it's just `<Navbar pathname={path} />`

Reply:
Lol that's what I tried, but I'm getting the same error `Property 'pathname' does not exist on type 'IntrinsicAttributes & Omit<NavBarProps, "logout" | "pathname">`, when I try that. I'm not sure if this line is correct: `const Navbar = connect(mapStateToProps, { logout })(UnconnectedNavbar);`

Reply:
oh yeah ur right you can remove both props, because the type is omitting logout and pathname

Reply:
could you do a quick search of all the places the navbar is being used and remove cartQuantity from them too? I forgot to comment this earlier

Reply:
Oh okay another thing you'll have to do for this unit test is mock the store router and set the pathname for each unit test since we aren't passing it in anymore